### PR TITLE
Basic main menu working with navigation and placeholder elements for future work

### PR DESCRIPTION
**Pull request by Kharjai**
Unfortunately, TextMeshPro required importing a whole bunch of base assets to be functional. They don't appear to be massive in size, but there are quite a few that clog this PR. Most of the functional changes are in the Scripts folder.

### CODE REVIEW

**Review by BrendenDomingo — COMMENTED** [review]
boo

File: Assets/Scripts/AssetImporter.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        {
+            if (gameObject.name.ToLower().Contains("mainmenubackground"))
+            {
+                var image = gameObject.GetComponent<Image>();
+                image.sprite = _assetController.MainMenuBackgroundSprite;
+                image.enabled = true;
+                continue;
+            }
+
+            var sliderComponent = gameObject.GetComponent<Slider>();
```
[the remark this answers is not in the document]
Comment (Kharjai) [reply Assets/Scripts/AssetImporter.cs:23]: The purpose of each of the game object components being captured in start is to edit their properties - we don't actually need to store them anywhere after init.

File: Assets/Scripts/AssetImporter.cs [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+using UnityEngine;
+using UnityEngine.UI;
+using TMPro;
+
+public class AssetImporter : MonoBehaviour
+{
+    [SerializeField] private List<GameObject> _spriteReferences;
+    [SerializeField] private AssetController _assetController;
+
+    void Start()
```
[the remark this answers is not in the document]
Comment (Kharjai) [reply Assets/Scripts/AssetImporter.cs:11]: Moved scoped code to private functions for readability.

File: Assets/Scripts/TitleRenderer.cs [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+    void Start()
+    {
+        _pressToContinueText = transform.Find("PressToContinueText").GetComponent<TextMeshProUGUI>();
+        _color = _pressToContinueText.color;
+        _fadeIn = true;
+        _fadeOut = false;
+    }
+
+    void FixedUpdate()
```
[the remark this answers is not in the document]
Comment (Kharjai) [reply Assets/Scripts/TitleRenderer.cs:23]: Check this comment against the current code to see if it is still relevant - not sure this was made against current rev

[the remark this answers is not in the document]
Comment (Kharjai) [reply Assets/Scripts/TitleRenderer.cs:23]: nvm this is neat

**Review by BrendenDomingo — APPROVED** [review]
sicko

**Pull request merged**